Documentation/Cluster.rst

```diff
@@ -50,18 +49,23 @@ Joining the node to the cluster
 
 **Note:** The following can only be performed by a superuser.
 
-**Note:** Both nodes must allow root login over SSH from the network chosen for MCVirt clustering.
+1. From the remote node, run:
+
+  ::
+
+    mcvirt cluster get-connect-string
 
-**Note:** ``/root/.ssh/known_hosts`` must exist to add a node to the cluster.
+The connect string will be displayed
 
-1. From the source node, run:
+2. From the source node, run:
 
   ::
-    
-    sudo mcvirt cluster add-node --node <Remote Node Name> --ip-address <Remote Cluster IP Address>
-    
 
-2. A prompt for the root password of the remote node will be presented.
+    mcvirt cluster add-node --connect-string <connect string>
+
+where ``<connect string>`` is the string printed out in step 1.
+
+
 3. The local node will connect to the remote node, ensure it is suitable as a remote node, setup authentication between the nodes and copy the local permissions/network/virtual machine configurations to the remote node.

```

Comment:
Could you put a note mentioning that all data on the remote node (e.g. VMs, users, permissions etc) will be removed whilst it is added to a cluster?